mypy/types.py

```diff
@@ -43,25 +45,74 @@ def deserialize(cls, data: JsonDict) -> 'Type':
         raise NotImplementedError('unexpected .class {}'.format(classname))
 
 
+class TypeVarId:
+    # 1, 2, ... for type-related, -1, ... for function-related
+    raw_id = 0  # type: int
+
+    # Level of the variable in type inference. Currently either 0 for
+    # declared types, or 1 for type inference unification variables.
+    meta_level = 0  # type: int
+
+    # Used for allocating fresh ids
+    next_raw_id = 1  # type: int
```

Comment:
It's worth mentioning this in a comment, as the current comment is a little misleading.